app/editor/package.json

```diff
@@ -59,7 +59,7 @@
     "redux-logger": "3.0.6",
     "styled-components": "6.1.11",
     "stylis": "4.3.2",
-    "tno-core": "0.1.98"
+    "tno-core": "./tno-core-0.1.98.tgz"
```

Comment:
All the changes to `/npm/core` will have to be published to a new version of `tno-core`. I can do that for you if you do not have access! Can check by running `make publish` from your `/npm/core` dir.